cloudflare/resource_cloudflare_zone_test.go

```diff
@@ -73,6 +73,28 @@ func TestAccCloudflareZonePartialSetup(t *testing.T) {
 	})
 }
 
+func TestAccCloudflareZonePartialSetupEnterprise(t *testing.T) {
```

Comment:
This test currently fails as the `plan` isn't enterprise but free

```
------- Stdout: -------
=== RUN   TestAccCloudflareZonePartialSetupEnterprise
--- FAIL: TestAccCloudflareZonePartialSetupEnterprise (1.62s)
    testing.go:569: Step 0 error: Check failed: Check 3/5 error: cloudflare_zone.tf-acc-partial-setup-zone: Attribute 'plan' expected "enterprise", got "free"
FAIL
```

I think this is related to #488 though

Reply:
uh ok, will look into that issue
does Travis run the acceptance tests?

Reply:
Afraid not; it's kicked off by TeamCity which has limited maintainer access and doesn't report here 🙁 

You can however, run these from your local machine using some [instructions from the README](https://github.com/terraform-providers/terraform-provider-cloudflare#developing-the-provider)

Reply:
Ok thanks, I was indeed able to see that fail using our test account... I suspect fixing that involves a change in the core of API?

Reply:
I’ve opened #511 to hopefully fix it. Stay tuned!